src/main/scala/wdl4s/WdlNamespace.scala

```diff
@@ -81,8 +82,9 @@ case class NamespaceWithWorkflow(importedAs: Option[String],
         optionValue = tryValue.get if tryValue.get.isDefined
       } yield key -> optionValue.get)
     } else {
-      val message = failures.values.collect { case f: Failure[_] => f.exception.getMessage }.mkString("\n")
-      Failure(new UnsatisfiedInputsException(s"The following errors occurred while processing your inputs:\n\n$message"))
+      val errors = failures.values.collect { case f: Failure[_] => f.exception.getMessage }
```

Comment:
In this particular case I don't think so because the only kind of exception possibly thrown during coercion is `UnsatisfiedInputException` which is not a `ThrowableWithErrors`.